span-processing-utils/src/main/java/org/hypertrace/config/span/processing/utils/SpanFilterMatcher.java

```diff
@@ -111,7 +111,13 @@ private boolean matches(String lhs, String rhs, RelationalOperator relationalOpe
       case RELATIONAL_OPERATOR_ENDS_WITH:
         return lhs.endsWith(rhs);
       case RELATIONAL_OPERATOR_REGEX_MATCH:
-        return Pattern.compile(rhs).matcher(lhs).find();
+        try {
```

Comment:
We would also have to upgrade the library version in all the consumers, since this is a utils class